locallens_api/locallens_api/products/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from .views import ProductListCreateView
+from .views import ProductRetrieveUpdateDestroyView
+
+urlpatterns = [
+    path("/", ProductListCreateView.as_view(), name="product_list_create"),
+    path(
+        "<int:pk>/",
+        ProductRetrieveUpdateDestroyView.as_view(),
+        name="product_retrieve_update_destroy",
+    ),
+]
```

Comment:
This code does not work, because there is no ProductListCreateView and ProductRetrieveUpdateDestroyView in views.py, which cause the application to crash

Reply:
we need to map the endpoints created to the specific url path